steam_deploy.sh

```diff
@@ -6,12 +6,18 @@ echo "#   Generating Depot Manifests  #"
 echo "#################################"
 echo ""
 
+if [ -n "$firstDepotIdOverride" ]; then
+  firstDepotId=$firstDepotIdOverride
+else
+  firstDepotId=$((appId + 1))
+fi
+
 i=1;
 export DEPOTS="\n  "
 until [ $i -gt 9 ]; do
   eval "currentDepotPath=\$depot${i}Path"
   if [ -n "$currentDepotPath" ]; then
-    currentDepot=$((appId+i))
+    currentDepot=$((firstDepotId + i - 1))
```

Comment:
I'm not sure but this doesn't look correct to me, when looking at the changes from both this line and 2 lines above.

Could you please doublecheck if this works as intended? If so, perhaps it might be useful to start adding a comment or two.

Reply:
Resolving as this comment has somehow been duplicated and I'm dealing with the other copy

Reply:
Ah yes, sorry I was dealing with temporary connection issues.